maps/biter_battles_v2/difficulty_vote.lua

```diff
@@ -114,11 +114,47 @@ local function on_player_left_game(event)
 	set_difficulty()
 end
 
+local function set_reroll_map_voting_status(player)
+	local yes = 0
+	local no = 0
+	for _, players in pairs(global.reroll_map_voting) do
+		if players == 1 then 
```

Comment:
```suggestion
	for _, vote in pairs(global.reroll_map_voting) do
		if vote == 1 then 
```
`players` confusing name